master/internal/db/postgres_tasks_intg_test.go

```diff
@@ -85,6 +85,8 @@ func TestJobTaskAndAllocationAPI(t *testing.T) {
 		AgentLabel:   "something",
 		ResourcePool: "somethingelse",
 		StartTime:    time.Now().UTC().Truncate(time.Millisecond),
+		State:        1,
+		IsReady:      false,
 	}
 	err = db.AddAllocation(aIn)
```

Comment:
what i meant was, we should call update allocation right below here, not change `AddAllocation`'s code add all and `model.Allocation.State` and `model.Allocation.IsReady` both need to be pointers since the column is null. sorry i didn't catch this in review.